packages/core/src/signer/ckb/signerCkbMultisig.ts

```diff
@@ -0,0 +1,245 @@
+import { Address } from "../../address/index.js";
+import {
+  multisigMetadataFromPubkeys,
+  Script,
+  SinceLike,
+  Transaction,
+  TransactionLike,
+} from "../../ckb/index.js";
+import { CellDepInfo, Client, KnownScript } from "../../client/index.js";
+import { Hex, hexConcat, hexFrom, HexLike } from "../../hex/index.js";
+import { SignerCkbPrivateKey } from "./signerCkbPrivateKey.js";
+
+export interface MultisigInfo {
+  pubkeys: HexLike[];
```

Comment:
`MultisigInfo` was used as `Signer.multisig` member's type, so I don't think we should use `*Like` here. Considering adding a `MulsigInfoLike` type for arg types.